src/scripts/main.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+const wallSize = parseInt(window.getComputedStyle(wall).width);
+const spiderSize = parseInt(window.getComputedStyle(spider).width);
+const maxCoord = wallSize - (spiderSize / 2);
+
+wall.addEventListener('click', e => {
+  function getDistance(coord) {
+    return coord > maxCoord
+      ? maxCoord
+      : coord < spiderSize / 2
+        ? spiderSize / 2 : coord;
```

Comment:
```suggestion
        ? spiderSize / 2 
        : coord;
```
be consistent with formatting